pkg/config/doc.go

```diff
@@ -0,0 +1,5 @@
+// Package config manages DC/OS CLI configurations for your clusters.
```

Comment:
s/`manages DC/OS CLI configurations for your clusters.`/`manages CLI configurations for your DC/OS clusters.`